ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -88,6 +88,7 @@ function extract_ccoctl(){
     else
         chmod 775 /tmp/ccoctl
     fi
+    export PATH="$PATH"
```

Comment:
strange, here on top of `extract_ccoctl()`  PATH was already exported, and its on the same scope as this one.. 
and I don't see where is modified in between. wonder how this one can fail without it..

Reply:
This is strange for me, too. We found this issue in extract_oc function, I just do the same enhancement for extract_ccoctl together, though I still figure out the root cause yet. More details, refer to https://redhat-internal.slack.com/archives/C05EV34PSKF/p1711847894307029